tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/builtinfunctions/FreezeAndIsFrozenTest.java

```diff
@@ -436,6 +436,16 @@ public void testFrozenValueUpdatePanicWithCheckTrap() {
                             "failed to set element to json: message=modification not allowed on readonly value");
     }
 
+    @Test
+    public void testXMLItemsCloneReadOnly() {
+        BValue[] returns = BRunUtil.invoke(result, "testXMLItemsCloneReadOnly", new BValue[0]);
+        Assert.assertTrue(((BBoolean) returns[0]).booleanValue());
+        Assert.assertTrue(((BBoolean) returns[1]).booleanValue());
+        Assert.assertTrue(((BBoolean) returns[2]).booleanValue());
+        Assert.assertTrue(((BBoolean) returns[3]).booleanValue());
+        Assert.assertTrue(((BBoolean) returns[4]).booleanValue());
```

Comment:
Shall we move these assertions to the bal file instead?

Reply:
Moved